From c2b485cddfa5b9b0c37cadc1fc43228b7e3308c0 Mon Sep 17 00:00:00 2001
From: Maxim Ostapenko <maxim.o@samsung.com>
Date: Tue, 26 Sep 2023 07:27:15 +0000
Subject: [PATCH 128/128] Revert "[gloo] Update to include patch for linux 6.0
 compatibility."

The 'https://github.com/cdluminate/gloo' repo became private so we can
not clone it anymore. We don't use GLOO anyways though so it's enough to
just revert the previous fix.

This reverts commit 4d675492e380daf2b7acc9cc29d4af1d50df85a8.

Signed-off-by: Maxim Ostapenko <maxim.o@samsung.com>
---
 .gitmodules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.gitmodules b/.gitmodules
index 057699f2..282746ed 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -33,7 +33,7 @@
 [submodule "third_party/gloo"]
     ignore = dirty
     path = third_party/gloo
-    url = https://github.com/cdluminate/gloo
+    url = https://github.com/facebookincubator/gloo
 [submodule "third_party/NNPACK_deps/pthreadpool"]
     ignore = dirty
     path = third_party/pthreadpool
-- 
2.34.1

